[2024-11-15T20:39:06.776+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-15T20:39:07.111+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: FLOAT_DATA_TO_PG.transform_data manual__2024-11-15T20:20:16.923650+00:00 [queued]>
[2024-11-15T20:39:07.134+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: FLOAT_DATA_TO_PG.transform_data manual__2024-11-15T20:20:16.923650+00:00 [queued]>
[2024-11-15T20:39:07.135+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-11-15T20:39:07.174+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-11-15 20:20:16.923650+00:00
[2024-11-15T20:39:07.188+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=503) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-15T20:39:07.193+0000] {standard_task_runner.py:63} INFO - Started process 505 to run task
[2024-11-15T20:39:07.193+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'FLOAT_DATA_TO_PG', 'transform_data', 'manual__2024-11-15T20:20:16.923650+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/etl_process_float_data.py', '--cfg-path', '/tmp/tmpuzv23_3u']
[2024-11-15T20:39:07.201+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask transform_data
[2024-11-15T20:39:07.366+0000] {task_command.py:426} INFO - Running <TaskInstance: FLOAT_DATA_TO_PG.transform_data manual__2024-11-15T20:20:16.923650+00:00 [running]> on host 35eb7317174e
[2024-11-15T20:39:07.807+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='FLOAT_DATA_TO_PG' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-15T20:20:16.923650+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-15T20:20:16.923650+00:00'
[2024-11-15T20:39:07.816+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-15T20:39:08.001+0000] {logging_mixin.py:188} INFO - Transformed DataFrame:
[2024-11-15T20:39:08.072+0000] {logging_mixin.py:188} INFO -      Client              Project  ...    End Date Estimated Hours
0  Client 1  Website Development  ...  2023-07-24             112
1  Client 1  Website Development  ...  2023-07-24              24
2  Client 1  Website Development  ...  2023-08-28             189
3  Client 1  Website Development  ...  2023-09-04              77
4  Client 1  Website Development  ...  2023-09-04              92

[5 rows x 8 columns]
[2024-11-15T20:39:08.131+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-15T20:39:08.132+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-15T20:39:08.164+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=FLOAT_DATA_TO_PG, task_id=transform_data, run_id=manual__2024-11-15T20:20:16.923650+00:00, execution_date=20241115T202016, start_date=20241115T203907, end_date=20241115T203908
[2024-11-15T20:39:08.247+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-15T20:39:08.319+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-15T20:39:08.323+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
